.github/workflows/build-image.yml

```diff
@@ -0,0 +1,22 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Build and Publish the Docker image
```

Comment:
Can we simply Build on `pull_request` and on `push` to master?

We can probably Publish the image on `release` with separate CI action.

This will be similar to PyPi. We do the same there. only check install package on `pull request` and `push` but do release to PyPi only on release.

Reply:
Yes. It is possible to just build as well.